### ISSUE TITLE
Feature Request: Whirlwind Integration Best Practices

### ISSUE THREAD

**Issue by workflow** [xolvio/automated-testing-best-practices#12]
How would you weave the scripts used in this best practices guide together with https://github.com/xolvio/whirlwind ?

**Comment by SamHatoum** [comment]
Yes, coming soon, just waiting for a few more tests to build up and I want to integrate WW

**Comment by ebbe-brandstrup** [comment]
Looking forward to this also :+1:

**Comment by danborufka** [comment]
any news on this front? Thanks a bunch

**Comment by SamHatoum** [comment]
I haven't got around to it yet, I will at some point I promise!

**Comment by lgandecki** [comment]
This would be game-changing for us, our app load/login/logout process is a bit slow (there is a ton of data loaded after the login), so we tend to group some tests together to speed the tests up. 
If we could split them easily on the paralleled CI servers (like circleci), we could easily improve our end-to-end tests quality, a lot I imagine.

Let me know if we can help in any way :) Is there some coding that has to be done still? Or is it more a matter of setting up a working (meteor, chimp, whirlwind, circleci) example?

**Comment by SamHatoum** [comment]
Would love some help! It's just configuration to get this working. Let's have a hangout?

**Comment by lgandecki** [comment]
Sure. I'll slack you in a sec so we can schedule something.

**Comment by lgandecki** [comment]
Hey guys, 

We didn't get around configuring whirlwind yet, but there is a branch with parallelization done on circleci, you can check it out here:

https://github.com/lgandecki/automated-testing-best-practices/tree/withChimp0.31.0

I've submitted PR and I'm waiting for the guys to review it and merge, meanwhile feel free to play around with it, and let me know what you think.

We use very similar setup to parallelize end-to-end tests in our production Meteor app. 

Here is an example of a build with x4 parallelization: https://circleci.com/gh/lgandecki/automated-testing-best-practices/27 (the feature files were copied and pasted a few times with some numbers added in them so they are recognizable in the builds, like "As an Account Holder 4" in tab 1)
